src/zhinst/toolkit/driver/devices/__init__.py

```diff
@@ -6,10 +6,11 @@
 from zhinst.toolkit.driver.devices.pqsc import PQSC
 from zhinst.toolkit.driver.devices.shfqa import SHFQA
 from zhinst.toolkit.driver.devices.shfsg import SHFSG
-from zhinst.toolkit.driver.devices.shfqc import SHFQC
 from zhinst.toolkit.driver.devices.uhfli import UHFLI
 from zhinst.toolkit.driver.devices.uhfqa import UHFQA
 
+from zhinst.toolkit.driver.devices.shfqc import SHFQC  # isort:skip
+
 DeviceType = t.Union[BaseInstrument, HDAWG, PQSC, SHFQA, SHFSG, UHFLI, UHFQA]
```

Comment:
Could you fix my bug 😁 ... I missed adding SHFQC to the DeviceType

Reply:
Sure. #130